src/app/modules/submission/submissions.statsd-client.ts

```diff
@@ -0,0 +1,34 @@
+import { Tags } from 'hot-shots'
+import { ResponseMetadata } from 'shared/types'
+
+import { statsdClient } from '../../config/datadog-statsd-client'
+
+import { getNormalisedResponseTime } from './submission.utils'
+
+export const submissionsStatsdClient = statsdClient.childClient({
+  prefix: 'formsg.submissions.',
+})
+
+export const reportSubmissionResponseTime = (
+  responseMetadata: ResponseMetadata,
+  tags: Tags,
+) => {
+  // response Time
+  submissionsStatsdClient.distribution(
+    'responseTime',
+    responseMetadata.responseTimeMs,
+    1,
+    tags,
+  )
+
+  // normalised response Time
+  submissionsStatsdClient.distribution(
+    'normResponseTime',
```

Comment:
/nit, I would rename this one to:
```suggestion
    'responseTime.normalized',
```

so in autocomplete, the 2 metrics are side by side

```
formsg.submissions.responseTime
formsg.submissions.responseTime.normalized
```


I was considering 
```suggestion
    'responseTime.normalized10',
```

but it feels over-constrained, so no need...

Reply:
Thanks Tim! [bf8343a](https://github.com/opengovsg/FormSG/pull/6406/commits/bf8343a7ddb8bab2c8c17d2a6a37c1609c69f532)